README.md

```diff
@@ -86,9 +74,39 @@ Options that are passed to the LaunchDarkly JS client for additional configurati
 - [Secure Mode](https://docs.launchdarkly.com/docs/js-sdk-reference#section-secure-mode)
 - [Setting LaunchDarkly Enterprise URLs](https://github.com/launchdarkly/js-client/blob/master/src/index.js#L241-L243)
 
+##### SSR Support
```

Comment:
The feature description seems lost when it's nested inside the component definition.  Also the `h5` and h6` are tiny.  SSR deserves its own big heading.

It probably also deserves a link to https://github.com/TrueCar/react-launch-darkly/#ssr-support in some kind of feature list near the top of the README.  This lib is now awesome enough to merit a little self promotion (almost) above the fold.

Reply:
Good idea, will update that.